vscode-dotnet-runtime-library/src/Acquisition/VersionResolver.ts

```diff
@@ -135,10 +136,16 @@ export class VersionResolver implements IVersionResolver {
         Debugging.log(`Resolving the version: ${version}`, this.context.eventStream);
         this.validateVersionInput(version);
 
-        const matchingVersion = releases.filter((availableVersions : IDotnetVersion) => availableVersions.channelVersion === version);
+        // Search for the specific version
+        let matchingVersion = releases.filter((availableVersions : IDotnetVersion) => availableVersions.version === version);
+        // If a x.y version is given, just find that instead (which is how almost all requests are given atm)
+        if(!matchingVersion || matchingVersion.length < 1)
+        {
+            matchingVersion = releases.filter((availableVersions : IDotnetVersion) => availableVersions.channelVersion === version);
+        }
```

Comment:
I do like how the `matchingVersion` is handled here to search for specific version first before find the channel version.

Reply:
Thank you! :)